descriptors/ModuleDescriptor-template.json

```diff
@@ -795,6 +723,94 @@
         "orders.acquisitions-units-assignments.all",
         "orders.order-templates.all"
       ]
+    },
+    {
+      "permissionName": "modperms.orders.item.put",
```

Comment:
please set the permissions to visible= false. we don't want them to appear on UI for users